modules/dnn/src/op_inf_engine.cpp

```diff
@@ -1162,6 +1174,12 @@ bool isMyriadX()
      return myriadX;
 }
 
+bool isArmPlugin()
+{
+    static bool armPlugin = getInferenceEngineCPUType() == "ARM";
```

Comment:
Lets make this check robust.
We do not really need to check for "ARM" CPU type here (we know that in compile time).
We should check for "ARM COMPUTE" plugin instead.

isArmPlugin => isArmComputePlugin
"ARM" => "ARM_COMPUTE"